tests/test_plugin_importing.py

```diff
@@ -1,14 +1,2 @@
-# -*- coding: utf-8 -*-
-
-import unittest
-
-
-class ImportPluginsTest(unittest.TestCase):
-    def test_importing_command_import_wordpress(self):
-        import nikola.plugins.command.import_wordpress  # NOQA
-
-    def test_importing_compile_rest(self):
-        import nikola.plugins.compile.rest  # NOQA
-
-    def test_importing_plugin_compile_markdown(self):
-        import nikola.plugins.compile.markdown    # NOQA
+def test_importing_plugin_task_galleries():
```

Comment:
This could be merged with the equally short `test_commands.py`.

Reply:
Merged them into `test_plugins.py`.